model/src/main/proto/topic.proto

```diff
@@ -204,7 +204,7 @@ message PromotedStory {
   // The thumbnail that should be displayed for this promoted story.
   LessonThumbnail lesson_thumbnail = 9;
 
-  // The indicator to handle if story is completed.
+  // Indicates whether the topic containing this story is 'learned' (meaning at least one story was completed).
```

Comment:
This line is longer than 100 lines, please break it up into multiple comments.

Reply:
Done.